src/daria/image/image.py

```diff
@@ -78,9 +85,19 @@ def __init__(
         # Fetch image
         if isinstance(img, np.ndarray):
             self.img = img
+
+            # Come up with default metadata
             self.name = "Unnamed image"
+            self.timestamp = None
+
         elif isinstance(img, str):
-            self.img = cv2.imread(str(Path(img)))
+            pil_img = PIL_Image.open(img)
```

Comment:
Should this be str(Path(img))? I am not sure if this is the best way to do things. Currently I am playing around in windows to see how this works there.

Reply:
As you could notice above, I did not get, that this was how you were foreseeing using pathlib...

Reply:
It turns out (in contrary to cv2) that PIL actually allows Path as input argument. So based on your comment above, I assume, something like this would be most unviersal and at the same time short?
```suggestion
            pil_img = PIL_Image.open(Path(img))
```

Reply:
That looks very good! Let us try to have it like this whenever we use PIL then.